### ISSUE TITLE
Basic (novice) question: running examples

### ISSUE THREAD

**Issue by uhaile** [chillerlan/php-qrcode#24]
I have done:
step 1: get composer 
step 2: composer install (created all dirs etc)

What do I do now to run the example(s) in the tests directory?

Could someone provide a simple index.php that would use the package, and is located in the root dir of my server? The package is installed in php-qrcode-master directory, under the root dir.

Thanks very much.

**Comment by codemasher** [comment]
Hi! The examples assume you have a local webserver running (with the project folder in the webroot), so that you can open them like `http://localhost/php-qrcode/examples/html.php`. Others can be run directly in the console like `php /path/to/examples/text.php`. I use [PhpStorm](https://www.jetbrains.com/phpstorm/), which has both integrated and is very comfortable to use.

**Comment by uhaile** [comment]
Hi and thanks for your reply.

I dont have a local webserver, using a shared server, while working with a windows 10 machine. 
I do my editing with emacs, which automatically uploads changes to the server. 

To run I use www.mydomain.com/mymainfile.php from any browser.

Is it that I need to include the following lines in my mymainfile.php:
// Autoload files using the Composer autoloader.
require_once __DIR__ . '/../vendor/autoload.php';

**Comment by codemasher** [comment]
Yes, change the directory to the destination of the `/vendor/autoload.php` of your installation.

**Comment by uhaile** [comment]
Thanks again for your reply.

Do you have a PHP file, which is not included in the package, which calls and demonstrates some of the functions of it? Something that will reside outside the directory tree of the package and all its dependencies created/maintained by composer.

I understand that whatever I write using the package should be in a separate directory, so that there is no confusion with the files maintained by composer. Is this correct?

**Comment by codemasher** [comment]
The example under [`/public`](https://github.com/chillerlan/php-qrcode/tree/master/public) is exactly the one that's running under https://smiley.codes/qrcode/ - if that doesn't help, i can't help you.
